live/development/terragrunt.hcl

```diff
@@ -10,7 +10,7 @@ terraform {
 
 # Configure input values for the specific environment being deployed:
 inputs = {
-  region = "us-west-2"
+  region = "us-east-1"
 
-  environment = "dev"
+  environment = "test"
```

Comment:
Just to ask, this is setting the `environment` to `test` in the `development` folder. That seems a little off. Any reason to not fork off a `test` terragrunt folder?

Reply:
Fixed in a later commit: https://github.com/Dozuki/CloudPrem-Infra/pull/3/commits/63531b68d368347d8a99db346988bdc811b2da7f